### PR TITLE
added Hulu clone

### PR DESCRIPTION
**Pull request by arsheelsheikh**
Added Hulu clone

I wish to add a Hulu clone to projects under Hacktoberfest 2021

This PR is an issue related to https://github.com/ZeroOctave/ZeroOctave-Javascript-Projects/issues/118

### CODE REVIEW

**Comment by arsheelsheikh** [comment]
@Astrodevil Done with the file name changes 

Here are the screenshots of the running project-

Screenshot 2021-10-13 at 11 48 29 AM

-

**Comment by arsheelsheikh** [comment]
> You are running whole project in a folder check left side of my screenshot. It's directory of this repo.

yes I get it but no idea why it's not running in the directory of the repo

**Comment by Astrodevil** [comment]
> 
> 
> > You are running whole project in a folder check left side of my screenshot. It's directory of this repo.
> 
> yes I get it but no idea why it's not running in the directory of the repo

because your linkage is wrong according to repo file location.

**Comment by arsheelsheikh** [comment]
> Any Changes?

I checked it, can you please help me with it!

**Comment by Astrodevil** [comment]
> 
> 
> > Any Changes?
> 
> I checked it, can you please help me with it!

you linkage is wrong. check them for every file in html.

File: Public/hulu.html [hunk 1]
```diff
@@ -174,10 +174,10 @@ <h4>Hulu + Live TV</h4>
...
 
         <div class="social-icons">
-          <a href="#"><img src="img/facebook.svg" alt="" /></a>
-          <a href="#"><img src="img/twitter.svg" alt="" /></a>
-          <a href="#"><img src="img/youtube.svg" alt="" /></a>
-          <a href="#"><img src="img/instagram.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-facebook.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-twitter.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-youtube.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-instagram.svg" alt="" /></a>
```
Comment (Astrodevil) [inline Public/hulu.html:180]: Like this one,  **assets/Images/hulucloneimg/instagram.svg**   is the correct address according to your commits but you did it wrong. 
you changed the svg image name in html file but not the file name of original svg, also you didn't specify the folder.

**Review by Astrodevil — COMMENTED** [review]
Still not working. Please do it properly, first clone this repo in your machine and put your project related files in respective directory of that cloned repo. Open whole folder in VS code and open your hulu html file and launch live server and make changes if anything is not working..also check linkage.

**Comment by Astrodevil** [comment]
> 
> 
> > **I am getting this**
> > Screenshot 2021-10-13 124624
> 
> no idea I see CSS is not working in your system

**See here your whole folder should look like thisVs code** look left side of vs code screen where evry file is opened.

**Comment by arsheelsheikh** [comment]
@Astrodevil I tried it same in my machine with the repository folder but it's showing the same
also, I made changes as suggested by you

**Comment by arsheelsheikh** [comment]
> @Astrodevil I tried it same in my machine with the repository folder but it's showing the same also, I made changes as suggested by you

can you edit them from your end and I'll commit the suggestion

**Comment by NOiR-07** [comment]
@arsheelsheikh I think you have deleted the branch from your forked repository, i suggest you to fork again and add your project and make a new PR

**Review by NOiR-07 — COMMENTED** [review]
i suggest you to fork again and add your project and make a new PR

**Comment by arsheelsheikh** [comment]
> i suggest you to fork again and add your project and make a new PR

Yup I guess that will work
trying now

**Comment by arsheelsheikh** [comment]
This PR can be closed in reference to https://github.com/ZeroOctave/ZeroOctave-Javascript-Projects/pull/132

**Pull request closed without merging**